src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Runtimes/MissingSdkRuntimeDetector.cs

```diff
@@ -1,7 +1,5 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
-using Microsoft.VisualStudio.ProjectSystem.VS;
```

Comment:
Can we somehow ban use of CPS's VS layer in our non-VS layer?

https://github.com/dotnet/roslyn-analyzers/pull/2161 seems like it might have helped here.

Reply:
At the least we could ban that extension method in our VS layer.

Reply:
I believe the CPS NuGet packages are going to be split up such that we won't automatically get a reference to the assemblies in each layer. That will make it easier to prevent this sort of thing--we just won't reference the VS layer from our non-VS layer.